components/html-block/html-block.js

```diff
@@ -194,11 +194,17 @@ class HtmlBlock extends RtlMixin(LitElement) {
 		this.noDeferredRendering = false;
 		this._hasSlottedContent = false;
 
+		this._contextObserverControllerResolve = undefined;
+		this._contextObserverControllerLoaded = new Promise(resolve => {
+			this._contextObserverControllerResolve = resolve;
+		});
+
 		getRenderers().then(renderers => renderers.reduce((attrs, currentRenderer) => {
 			if (currentRenderer.contextAttributes) currentRenderer.contextAttributes.forEach(attr => attrs.push(attr));
 			return attrs;
 		}, [])).then(rendererContextAttributes => {
 			this._contextObserverController = new HtmlAttributeObserverController(this, ...rendererContextAttributes);
```

Comment:
I could have wrapped this in a promise directly, but I think it's still ideal if the renderer promise starts resolving ASAP, not just when we first try to access the controller.